server/tserver/src/main/java/org/apache/accumulo/tserver/TabletServer.java

```diff
@@ -2310,6 +2310,8 @@ private HostAndPort startReplicationService() throws UnknownHostException {
   }
 
   public ZooLock getLock() {
+    log.info("Acquiring Tablet Server Lock");
+    log.info("Lock Acquired");
```

Comment:
I'm not sure this is the correct way to do this, but the TabletServer lock is clearly different from the other ones.